test/index.test.js

```diff
@@ -31,10 +48,42 @@ describe('figmadiffer', () => {
       }
     }
 
-    figmadiffer(robot)
+    // Mock the Figma API
+    nock('https://api.figma.com/v1')
```

Comment:
🤘 I was having trouble with this using the `figmadiffer` so I just commented it out before.